sdv/multi_table/base.py

```diff
@@ -429,33 +429,32 @@ def get_constraints(self):
 
         return constraints
 
-    def load_custom_constraint_classes(self, table_name, filepath, class_names):
+    def load_custom_constraint_classes(self, filepath, class_names):
         """Load a custom constraint class for the specified table's synthesizer.
```

Comment:
nitpick: should be changed to something like "Load a custom constraint class for each table's synthesizer"